add-domain

```diff
@@ -1,6 +1,7 @@
 www.tatasteelindustries.co.in
 109.197.125.34.bc.googleusercontent.com
 2020logs.duckdns.org
+https://coinbase-secure-help.com/
```

Comment:
We only put domains here. Please remove `https://` and the trailing `/`. Thanks!